src/main/java/com/instana/operator/AgentDeployer.java

```diff
@@ -71,7 +49,7 @@
 import static com.instana.operator.util.ResourceUtils.hasOwner;
 import static com.instana.operator.util.ResourceUtils.hasSameName;
 import static com.instana.operator.util.ResourceUtils.name;
-import static com.instana.operator.util.StringUtils.isBlank;
+import static com.instana.operator.util.StringUtils.*;
```

Comment:
We prefer not to use 'star' imports. For bigger lists we have less of an issue with that, but for one or two imports please reference these directly.

Reply:
Ok, its changed.